### PR TITLE
About

### PR DESCRIPTION
**Pull request by thekarinaten**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -49,8 +76,28 @@ button {
   white-space: nowrap;
 }
 
-.promo-section {
-  padding-bottom: 130px;
+.content-title {
+  color: rgb(255, 255, 255);
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  color: #fff;
```

File: assets/css/style.css [hunk 2]
```diff
@@ -49,8 +76,28 @@ button {
...
+  font-family: "Yeseva One";
```
Comment (katsu223) [inline assets/css/style.css]: Fallback font

File: assets/css/style.css [hunk 3]
```diff
@@ -49,8 +76,28 @@ button {
...
+  font-size: 90px;
+  font-weight: 100;
+  max-width: 970px;
+  padding-top: 130px;
+  text-align: center;
+  margin: 0px auto;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  margin: 0 auto;
```

**Review by katsu223 — CHANGES_REQUESTED** [review]

File: assets/css/style.css [hunk 4]
```diff
@@ -3,6 +3,33 @@
...
+.about-food {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  max-width: 430px;
+  text-align: center;
+}
+
+.about-title {
+  color: #cc9900;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  color: #c90;
```

File: assets/css/style.css [hunk 5]
```diff
@@ -49,8 +76,28 @@ button {
...
+  font-weight: 100;
+  max-width: 970px;
+  padding-top: 130px;
+  text-align: center;
+  margin: 0px auto;
+}
+
+div {
+  box-sizing: border-box;
+}
```
Comment (katsu223) [inline assets/css/style.css]: Move to the top of file

File: assets/css/style.css [hunk 6]
```diff
@@ -49,8 +76,28 @@ button {
...
+
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 36px;
+
 }
```
Comment (katsu223) [inline assets/css/style.css]: Do not use tags for styling, and smth wrong with braces

**Review by katsu223 — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 4 above)
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  color: #c90;
```

File: assets/css/style.css [hunk 7]
```diff
@@ -22,7 +49,7 @@ button {
   justify-content: center;
   max-width: 486px;
   margin: 0 auto;
-  padding-bottom: 130px;
+  box-sizing: border-box;
```
Comment (katsu223) [inline assets/css/style.css]: ?

File: assets/css/style.css [hunk 8]
```diff
@@ -3,6 +3,33 @@
...
+.about-txt {
+  color: #4c4c4c;
+  font-family: 'Lato', 'Arial', 'Times New Roman', sans-serif;
+  font-size: 16px;
+  padding-top: 30px;
+}
+
 body {
   margin: 0 auto;
 }
```
Comment (katsu223) [inline assets/css/style.css:40]: Move body below .class slectors

File: assets/css/style.css [hunk 9]
```diff
@@ -122,5 +185,4 @@ button {
 .wrapper {
   max-width: 1600px;
   margin: 0 auto;
-}
-
+}
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
}

```
and place .wrapper under the body

**Review by katsu223 — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 8 above)
Comment (thekarinaten) [reply assets/css/style.css:40]: below or above

**Review by katsu223 — APPROVED** [review]

**Pull request merged**